python_ta/typecheck/base.py

```diff
@@ -268,6 +268,10 @@ def can_unify(self, t1, t2):
                 return True
             else:
                 return False
+        elif isinstance(t1, GenericMeta):
```

Comment:
I originally put this in because isinstance raises an error when called on parametrized generics. I ran into the same problem with issubclass so I decided to put the check back in, which seems to handle the problem.